docs/ref/contrib/auth.txt

```diff
@@ -629,6 +629,16 @@ The following backends are available in :mod:`django.contrib.auth.backends`:
    users because :attr:`~RemoteUserBackend.user_can_authenticate` always
    returns ``True``.
 
+.. class:: DefaultObjectPermissionsBackend
```

Comment:
Again, I'd put these changed next to `ModelBackend`.  (`RemoteUserBackend` is pretty niche...) 

* I think in this block you need to include the `AUTHENTICATION_BACKENDS` settings example to show how to actually configure use this: it's important that it's used **in addition** to `ModelBackend`
* The `superset` phrasing assumes too much knowledge: We need an example of the behaviour with the default settings and adding the new backend. Something **along the lines of** "with `DefaultObjectPermissionsBackend` the permission check will fallback to the general permission if the user doesn't have the specific object permission".